src/features/calendar/components/CalendarWeekView/HeaderWeekNumber.tsx

```diff
@@ -0,0 +1,27 @@
+import { Msg } from 'core/i18n';
+import theme from 'theme';
+import { Box, Typography } from '@mui/material';
+
+import messageIds from '../../l10n/messageIds';
+
+type CalendarWeekNumberProps = {
+  weekNr: number;
+};
+
+const HeaderWeekNumber = ({ weekNr }: CalendarWeekNumberProps) => {
+  return (
+    <Box alignItems="center" display="flex" marginTop="2px">
+      <Typography
+        color={theme.palette.secondary.light}
+        fontStyle="bold"
+        sx={{ fontWeight: 800 }}
```

Comment:
I think you can remove these two lines, using `variant='subititle2'` does this for you!